### ISSUE TITLE
jestpack-loader issue

### ISSUE THREAD

**Issue by creaux** [webpack-contrib/npm-install-webpack-plugin#66]
Hello,

added jestpack into my build and it for some reason trying to install by name which doesn't exists.

Jestpack is in npm repository as `jestpack` and not as `jestpack-loader` which is trying to be installed.

``` bash
Installing jestpack-loader...
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.3.1/bin/node" "/usr/local/bin/npm" "install" "jestpack-loader" "--save"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/jestpack-loader
npm ERR! 404
npm ERR! 404  'jestpack-loader' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pjuna/Development/javascript/redux/pyxis_on_redux/npm-debug.log
```

Is there any way of ignoring specific packages? I see this solution as quick possible override. 

Thank you

**Comment by ericclemmons** [comment]
@creaux Thanks for bringing this up!

This sounds similar to #61, in which I proposed the following:
> https://github.com/ericclemmons/npm-install-webpack-plugin/issues/61#issuecomment-233202234

**tldr; If you already have `jestpack` installed, then this would be skipped.** (Once #61 is fixed :D)

The dilemma here (which puts automation in a tough spot) is how `babel` throws cryptic errors when tried as a loader, which is why Webpack (and this project) appends loaders with `-loader` to ensure `babel-loader` is treated separately from `babel`.

**Comment by creaux** [comment]
So closing regarding #61
